server/src/main/java/fr/uga/l3miage/tp1/exo3/models/AssociationEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp1.exo3.models;
+
+import fr.uga.l3miage.tp1.exo3.enums.AssociationType;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class AssociationEntity {
+    @Id
+    private String name;
+    @NotNull
+    private String presidentName;
+    @NotNull
+    private String treasureName;
```

Comment:
alos pas bête l'utilisation du `@NotNull`, cependant ici c'est au niveau du code java que ça s'applique. Moi je voulais ça en table : 
```suggestion
    @NotNull
    @Column(nullable = false)
    private String presidentName;
    @NotNull
     @Column(nullable = false)
    private String treasureName;
```